packages/iframe-execution-environment-service/src/IframeExecutionEnvironmentService.ts

```diff
@@ -150,8 +150,10 @@ export class IframeExecutionEnvironmentService
 
     Object.values(jobWrapper.streams).forEach((stream) => {
       try {
-        !stream.destroyed && stream.destroy();
-        stream.removeAllListeners();
+        if (stream && !stream.destroyed) {
+          stream.destroy();
+          stream.removeAllListeners();
+        }
```

Comment:
After converting the interfaces in here to object types, I got build errors for these lines because the stream could be `null`, which is in fact true. Another reason why interfaces suck!!